[3.2.0] Cannot find module './lib/axios'

<!--
Thanks for reporting an issue to axe-core. Please provide all necessary info to reproduce the issue. Without adequate details, your issue may be closed without investigation.

If you’re reporting a bug, include a description of the issue and a page or code snippet where it can be reproduced. Please make sure you have tested this with the latest version of axe-core. When proposing a new rule, please use our rule template: https://github.com/dequelabs/axe-core/blob/develop/doc/rule-proposal.md

Describe what the desired behavior would be, and how it applies to axe-core's  "Accessibility Supported" policy: https://github.com/dequelabs/axe-core/blob/develop/doc/accessibility-supported.md
-->
Attempts to update `axe-core` from 3.1.2 to 3.2.0 are causing Cypress tests to fail. From the error message, it looks like it can't resolve `axios` anymore? 

Expectation: no errors

Actual: `Error: Cannot find module './lib/axios' from '/root/project/node_modules/axe-core'`

Motivation: N/A

<pre><code>
axe-core version: 3.2.0
axe-webdriver, extension or other integration version: N/A

Browser and Assistive Technology versions

For Tooling issues:
- Node version: 10  <!-- run `node --version` -->
- Platform: all <!-- Mac, Linux, Windows -->
</code></pre>

Here's a reproduction using `create-react-app` and Cypress (test copied from #755):

https://github.com/wKovacs64/cypress-axe-core-320

(Run `yarn cy:run` to start Cypress in headless mode in your CLI or `yarn cy:open` to launch the GUI.)

Thanks.

--- a/build/imports-generator.js
+++ b/build/imports-generator.js
@@ -1,5 +1,6 @@
 const path = require('path');
 const browserify = require('browserify');
+const derequire = require('derequire');
 const createFile = require('./shared/create-file');
 
 const inputFile = path.join(
@@ -25,6 +26,8 @@ async function run() {
 			throw new Error('Cannot browserify axe.imports', err);
 		}
 		try {
+			// Replace `require` calls with `_dereq_` in order not to confuse Cypress.js
+			result = derequire(result);
 			await createFile(outputFile, result);
 		} catch (error) {
 			throw new Error('Cannot write browserify generated axe.imports', error);

--- a/package.json
+++ b/package.json
@@ -85,6 +85,7 @@
 		"chai": "~4.1.2",
 		"clone": "~2.1.1",
 		"css-selector-parser": "^1.3.0",
+		"derequire": "^2.0.6",
 		"dot": "~1.1.2",
 		"emoji-regex": "7.0.3",
 		"es6-promise": "^4.2.6",

--- a/test/integration/full/umd/umd-module-exports.js
+++ b/test/integration/full/umd/umd-module-exports.js
@@ -6,6 +6,16 @@ describe('UMD module.export', function() {
 		assert.strictEqual(module.exports, axe);
 	});
 
+	it('does not use `require` functions', function() {
+		// This is to avoid colliding with Cypress.js which overloads all
+		// uses of variables named `require`.
+		assert.notMatch(
+			axe.source,
+			/[^.]require\(/,
+			'Axe source should not contain `require` variables'
+		);
+	});
+
 	it('should ensure axe source includes axios', function() {
 		assert.isTrue(axe.source.includes(axe.imports.axios.toString()));
 	});

